src/chains/sui/Sui Report.md

```diff
@@ -49,20 +49,56 @@ The desirable properties related to Sui’s architecture and its integration wit
 
 ## Section 2: Network and Protocol Integrity [Common Prefix, Eiger, NodeMonster]
 ### 2.1 Network Architecture
-- Assessment of Sui's architecture, consensus mechanism, and staking requirements.
-- Assessment of the Sui team, governance structure, and decentralization status.
+Sui is a Layer 1 blockchain that supports scalable, high-performance decentralized applications. Unlike traditional blockchain architectures, Sui employs an object-based accounting model and a modified Delegated Proof-of-Stake (DPoS) consensus mechanism. Transactions are categorized into "simple" and "complex," with simple transactions bypassing traditional consensus steps, enhancing performance. Complex transactions leverage Narwhal for data availability and Bullshark for ordering, ensuring efficiency and resilience.
+
+Key Architectural Features:
+- **Object-Based Accounting Model:** Combines features of UTXO and account-based models to enable granular state management, making it particularly suitable for complex dApps.
+- **Consensus:** The Sui Network uses a Narwhal and Tusk consensus mechanism, which combines DAG-based mempool design (Narwhal) with a Byzantine Fault Tolerant (BFT) consensus algorithm (Tusk). Validators individually validate transactions and generate certificates of finality, optimizing for throughput and latency.
```

Comment:
```suggestion
- **Consensus:** The Sui Network uses a Narwhal and Tusk consensus mechanism, which combines DAG-based mempool design (Narwhal) with a Byzantine Fault Tolerant (BFT) consensus algorithm (Tusk). Validators individually validate transactions and generate certificates of finality, optimizing for throughput and latency. More specifically, Sui currently relies on the Mysticeti consensus algorithm, which uses Narwhal and Tusk as its basis. Mysticeti has been formalized in the form of [an academic research paper](https://arxiv.org/pdf/2310.14821) published on arXiv, with rigorous proofs of safety and liveness. Notably, Mysticeti achieves the optimal consensus latency of three network round trips, resulting in a ~4x latency reduction to Sui mainnet. Rather than relying on a single leader to propose a block, Mysticeti supports multiple validators proposing blocks in parallel, making use of the full bandwidth of the network. Moreover, Mysticeti finalizes transactions upon transaction inclusion, such that additional confirmations or network activity is not needed. Finally, Sui validators who attempt to censor valid transactions are accountable.
```

Reply:
Sui has switched to Mysticeti: https://sui.io/mysticeti